sale_procurement_group_by_line/model/sale.py

```diff
@@ -20,16 +20,14 @@ def _prepare_procurement_group_by_line(self, line):
 class SaleOrderLine(models.Model):
     _inherit = "sale.order.line"
 
-    @api.multi
     def _get_procurement_group_key(self):
         """ Return a key with priority to be used to regroup lines in multiple
         procurement groups
 
         """
         return 8, self.order_id.id
 
-    @api.multi
-    def _action_launch_stock_rule(self):
+    def _action_launch_stock_rule(self, previous_product_uom_qty=False):
```

Comment:
This method is overriding standard one, right? if so, can you add a comment stating that?

Reply:
well, it's calling super, so it's a bit obvious :P

Reply:
I'm a little bit confused, because there is a super at the end but the method seems to be a copy of the standard one with few changes, can you check what happens in supers, is the procurement executed twice?

Reply:
This super is a tricky way to maintain inheritance. Overriding methods can arise problems, that's why its better to call super if possible.

Reply:
@MiquelRForgeFlow Yes, I know that :laughing: My concern is that this call to super is execunting a almost same code just after. That's why I thought the method was overriding (meaning it has no call to super) but it is not, so the question is... should it be overriden? or can it be improved?

Reply:
But I already said that it should not be overridden because is preferable to have `super`. This method is also called in `procurement_jit` (which have auto_install=True), and many modules depend from `sale_stock` but not from `procurement_jit`, so it may arise problems if we don't put the `super`. And it cannot be improved because the method is complex and there is no way do it easier. It's not something of this migration, but something we have in previous versions and something I already checked before.

Reply:
@MiquelRForgeFlow you are not understanding my concern... Maybe I didn't explain myself correctly. Anyway, the issue has been discussed with @JoanSForgeFlow and it is clarified. Thanks for having a look!